modin/backends/base/query_compiler.py

```diff
@@ -239,204 +1170,443 @@ def to_numpy(self, **kwargs):
     # such that columns/rows that don't have an index on the other DataFrame
     # result in NaN values.
 
-    def add(self, other, **kwargs):
+    @_doc_binary_method(operation="addition", sign="+")
+    def add(self, other, **kwargs):  # noqa: PR02
```

Comment:
there is a [note](https://github.com/dchigarev/modin/blob/8d804b7d4a235ca00b680f8e813da6ced93be5a2/modin/backends/base/query_compiler.py#L87) about this at the top of the class. Shortly: the silenced warning indicates that there are some parameters documented in the doc-string that are not presented in the function signature, this happens because `**kwargs` hiding actual arguments that we want to document, the solution is to unpack `**kwargs`.